Mathlib/Tactic/Linter/EmptyLine.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2025 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Mathlib.Tactic.Linter.Header
+
+/-!
+#  The "emptyLine" linter
+
+The "emptyLine" linter emits a warning on empty lines inside a command, but outside of a
+doc-string/module-doc.
+-/
+
+open Lean Elab Linter
+
+/-- Retrieve the `String.Range` of a `Substring`. -/
+def Lean.Substring.getRange : Substring → String.Range
```

Comment:
Why is this in the `Lean` namespace; isn't `Substring` in the root namespace?

Reply:
You are right: I removed `Lean`!